source/utilities.tex

```diff
@@ -15581,6 +15644,215 @@
 Equivalent to: \tcode{return \exposid{underlying_}.format(elems, ctx);}
 \end{itemdescr}
 
+\indexlibraryglobal{\exposid{range-default-formatter}}%
+\begin{codeblock}
+namespace std {
+  template<ranges::@\libconcept{input_range}@ R, class charT>
+  struct @\exposid{range-default-formatter}@<range_format::map, R, charT> {
+  private:
+    using @\exposid{maybe-const-map}@ = @\exposid{fmt-maybe-const}@<R, charT>;                        // \expos
+    using @\exposid{element-type}@ =                                                      // \expos
+      remove_cvref_t<ranges::range_reference_t<@\exposid{maybe-const-map}@>>;
+    range_formatter<@\exposid{element-type}@, charT> @\exposid{underlying_}@;                         // \expos
+  public:
+    constexpr formatter();
```

Comment:
```suggestion
    constexpr @\exposid{range-default-formatter}@();
```

Reply:
Thanks, that clarifies my main issue with the paper :). done